go/vt/vterrors/constants.go

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2021 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package vterrors
+
+import "regexp"
+
+// Operation not allowed error
+const (
+	NotServing   = "operation not allowed in state NOT_SERVING"
+	ShuttingDown = "operation not allowed in state SHUTTING_DOWN"
+)
+
+// RxOp regex for operation not allowed error
+var RxOp = regexp.MustCompile("operation not allowed in state (NOT_SERVING|SHUTTING_DOWN)")
+
+// WrongTablet for invalid tablet type error
+const WrongTablet = "wrong tablet type"
+
+// RxWrongTablet regex for invalid tablet type error
+var RxWrongTablet = regexp.MustCompile(WrongTablet)
```

Comment:
error message cannot change as it would force an upgrade order. We should keep the error message same as what the tablet returns today. If we want to change the error message we would do it over 2 releases.

Reply:
can we change the error message and look for both the old and new message?

Reply:
could we change the message and check for both messages?